sdk/communication/Azure.Communication.PhoneNumbers/src/PhoneNumbersClient.cs

```diff
@@ -81,7 +81,7 @@ private PhoneNumbersClient(string endpoint, TokenCredential tokenCredential, Pho
         { }
 
         private PhoneNumbersClient(string endpoint, HttpPipeline httpPipeline, PhoneNumbersClientOptions options)
-            : this(new ClientDiagnostics(options), httpPipeline, endpoint, options.AcceptedLanguage, options.Version)
+            : this(new ClientDiagnostics(options, suppressNestedClientActivities: true), httpPipeline, endpoint, options.AcceptedLanguage, options.Version)
```

Comment:
This variable is used to prevent having duplicate scopes for what is essentially the same operation